server/protocols/jmap-integration-testing/rabbitmq-jmap-integration-testing/src/test/java/org/apache/james/jmap/rabbitmq/RabbitMQDeletedMessagesVaultTest.java

```diff
@@ -52,5 +56,18 @@ protected GuiceJamesServer createJmapServer() throws IOException {
             binder -> binder.bind(MailRepositoryDeletedMessageVault.Configuration.class)
                 .toInstance(new MailRepositoryDeletedMessageVault.Configuration(MailRepositoryUrl.from("cassandra://var/deletedMessages/user"))));
     }
+
+    @Override
+    protected void await() {
```

Comment:
it deserves a better name because it's not clear what it should await.

Reply:
`awaitSearchUpToDate` ?